src/renderers/dom/server/ReactServerRenderingTransaction.js

```diff
@@ -7,12 +7,14 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactServerRenderingTransaction
+ * @flow
```

Comment:
Flow does pass here. Adding an annotation to `ReactServerRenderingTransaction(renderToStaticMarkup: boolean)` was enough for flow.